Opserver/Controllers/GraphController.Spark.cs

```diff
@@ -63,10 +64,10 @@ public partial class GraphController
             if (iface == null) return ContentNotFound();
             var points = await iface.GetUtilization(SparkStart, null, SparkPoints);
 
-            Func<DoubleGraphPoint, double> getter = p => p.Value.GetValueOrDefault(0);
-            if (direction == "out") getter = p => p.BottomValue.GetValueOrDefault(0);
+            Func<DoubleGraphPoint, double?> getter = p => p.Value;
+            if (direction == "out") getter = p => p.BottomValue;
 
-            return SparkSVG(points, Convert.ToInt64(points.Max(getter)), getter);
+            return SparkSVG(points, Convert.ToInt64(points.Max(getter)), p => getter(p).GetValueOrDefault(0));
```

Comment:
Arf this is a last minute change. I initially use `? :` around `Convert.ToInt64` call .